.eslintrc.json

```diff
@@ -1,13 +1,20 @@
 {
-	"extends": "marine/prettier/node",
+	"extends": ["neon/common", "neon/node", "neon/typescript", "neon/prettier"],
 	"parser": "@typescript-eslint/parser",
 	"parserOptions": {
-		"project": "./tsconfig.eslint.json",
-		"extraFileExtensions": [".mjs"]
+		"project": "./tsconfig.eslint.json"
 	},
 	"plugins": ["local"],
 	"rules": {
+		"local/explicit-undefined-on-optional-properties": ["error", { "interfaceEndings": ["JSONBody"] }],
```

Comment:
This dupe seems wrong?

Reply:
Whilst I don't know the intricacies of the two local rules, both seem to exist in /.eslintplugin and seem to check for an explicit `undefined` on an optional property and a `?` on `undefined` properties respectively.

Reply:
Woops, misread the rule names 🥴